contracts/Makefile

```
@@ -0,0 +1,23 @@
+.PHONY: all deps compile clean generate-abi
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)